apps/api/src/prisma/prisma.service.ts

```diff
@@ -4,6 +4,25 @@ import { INestApplication, Injectable, OnModuleInit } from '@nestjs/common'
 @Injectable()
 export class PrismaService extends PrismaClient implements OnModuleInit {
   async onModuleInit() {
+
+    this.$use(async (params, next) => {
+      if (params.model == 'Expense') {
```

Comment:
This part must be more generic as it is part of the common `PrismaService` but it includes the specific `Expense` entity

Reply:
We don't want everything to be soft deleted.

There should be a list `string[]` of entities that support soft delete

Reply:
I think I understand the point now. So basically I should create an array with strings and manually write a model that needs to be soft deleted inside it , and check if the string[] includes the params.model and if it does then proceed with the soft delete. The question is where to create this array so everybody would find it easily?

Reply:
And I think I'm doing something wrong , because I restored the branch so I could make the changes you adviced me to , yet you deleted it , I thought after merging into master I should delete it , can you explain to me what's the proceedure in that case?